docs/guides/latest-updates.mdx

```diff
@@ -49,22 +49,26 @@ Explore the [v1.3 release notes](/api/qiskit/release-notes/1.3), and read about
 
 {/* use url format /api/qiskit/release-notes/X.X to link to the specific X.X version of the release notes */}
 
-### Qiskit Runtime client 0.28.0
+### Qiskit Runtime client 0.37.0
 
 *To see the release notes for all versions, visit the [Qiskit Runtime client release notes](/api/qiskit-ibm-runtime/release-notes).*
 
-**Highlights from the 0.28.0 (2024-08-15) version release**
+**Highlights from the 0.37.0 (13 March 2025) version release**
 
-#### Removal
+#### Deprecation
 
-- The V1 primitives `SamplerV1` and `EstimatorV1` have been completely removed. Refer to the [migration guide](/migration-guides/v2-primitives) for instructions on migrating to V2 primitives.
-- The `service` parameter is now required in `Session.from_id()`.
+The `IBMBackend` attributes `max_shots` and `max_experiments`, as well as the property `max_circuits` have all been deprecated. These attribute values used to represent the maximum number of shots and circuits that could be submitted in one job. See [job limits](/guides/job-limits#job-limits) for details.
```

Comment:
> These attribute values used to represent the maximum number of shots and circuits that could be submitted in one job.

Something like this, to remove ambiguity around "used" :
"These attribute values were used to represent" / "...had been used to represent" / "Formerly, these attributes represented..."